### PR TITLE
VAPI-1 Update AMD documentation

### PR DESCRIPTION
**Pull request by tmoney-bw**
## For the Committer

All PRs on this repo that change any API sources of truth (markdown files, OpenAPI specs) require a changelog added to the `site/src/pages/changelog.md` file. If this PR does not require changelog updates, you need to add the `no-changelog` tag to this PR before opening it.

Please confirm that you have either updated `site/src/pages/changelog.md` or added the `no-changelog` tag.

### CODE REVIEW

**Comment by nashley** [comment]
Related to https://github.com/Bandwidth/bandwidth.github.io/pull/719

File: site/docs/voice/webhooks/answer.mdx [hunk 1]
```diff
@@ -89,3 +90,24 @@ Content-Type: application/json
...
+    "applicationId"         : "7fc9698a-b04a-468b-9e8f-91238c0d0086",
+    "from"                  : "+15551112222",
+    "to"                    : "+15553334444",
+    "direction"             : "outbound",
+    "callId"                : "c-95ac8d6e-1a31c52e-b38f-4198-93c1-51633ec68f8d",
+    "callUrl"               : "https://voice.bandwidth.com/api/v2/accounts/55555555/calls/c-95ac8d6e-1a31c52e-b38f-4198-93c1-51633ec68f8d",
+    "startTime"             : "2019-06-20T15:54:22.234Z",
+    "answerTime"            : "2019-06-20T15:54:25.432Z",
+    "tag"                   : "example-tag",
+    "machineDetectionResult": "answering-machine"
```
Comment (nashley) [inline site/docs/voice/webhooks/answer.mdx:112]:
```suggestion
    "machineDetectionResult": {
        "value":  "answering-machine",
        "duration": "PT5.023724322S"
    }
```

**Review by nashley — CHANGES_REQUESTED** [review]

**Comment by nashley** [comment]
Links work. It's a shame we can't link directly to `machineDetection` in `POST /calls` anymore with the new doc site :/

**Review by nashley — APPROVED** [review]

**Review by nashley — CHANGES_REQUESTED** [review]
lol sorry for changing my mind again
`duration` should be a reasonable value given `startTime` and `eventTime`. We should probably just use a real ~callback~ webhook example.

**Review by nashley — APPROVED** [review]

**Review by nashley — CHANGES_REQUESTED** [review]
Bad json

**Review by nashley — APPROVED** [review]

**Review by vsmith-bw — APPROVED** [review]

**Pull request merged**